src/TextInput/TextInput.tsx

```diff
@@ -158,12 +157,13 @@ const Content = styled.div<IInputOutline>`
 
 const Input = styled.input<IInput>`
   border: none;
+  background-color: transparent;
   color: ${({ error }) => theme.colors[`${error ? 'error' : 'secondary'}`]};
   font-size: 16px;
   width: 100%;
   outline: none;
-  cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'pointer')};
-  opacity: ${({ disabled }) => (disabled ? '0.5' : '1')};
+  cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'initial')};
```

Comment:
I'm not sure why it was pointer but I feel like user expect the initial cursor of an input for an input?